### PR TITLE
creating an initial structure for our app

### PR DESCRIPTION
**Pull request by shlior7**
using djangos views to render the welcome_page.html now only showing "coming soon"

### CODE REVIEW

**Comment by adiabramovitch** [comment]
There are two empty files, is there a reason?

**Comment by ronyyosef** [comment]
image
looks great to me

**Review by Yoavsc129 — APPROVED** [review]

**Comment by adiabramovitch** [comment]
You shouldn't have two commits for this issue. 
For further notice you can:
 ```git commit --amend``` to add your current changes to the previous commit. 
Now you should rebase it and leave a proper commit message.

File: main/templates/main/index.html [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+  </div>
+  <div class="middle">
+    <h1>COMING SOON</h1>
+    <hr>
+    <p id="demo" style="font-size:30px"></p>
+  </div>
+  <div class="bottomleft">
+    <p>Some text</p>
+  </div>
+</div>
```
Comment (adiabramovitch) [inline main/templates/main/index.html:59]: Did you mean to leave 'Logo' and 'Some text'?

Comment (shlior7) [reply main/templates/main/index.html]: No i removed the wrong duplicate by mistake :)

**Pull request merged**